Tests/AlchemyDataNewsV1Tests/AlchemyDataNewsTests.swift

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  **/
 import XCTest
-import AlchemyDataNewsV1
 import Foundation
+import AlchemyDataNewsV1
```

Comment:
I like that you arranged some of the existing import statements to make the order consistent throughout the tests.